src/groups/rotation_action.jl

```diff
@@ -214,3 +214,76 @@ group_manifold(A::RowwiseMultiplicationAction) = A.manifold
 function inverse_apply(::LeftRowwiseMultiplicationAction, a, p)
     return (a \ p')'
 end
+
+###
+
+@doc raw"""
+    ColumnwiseMultiplicationAction{
+        TM<:AbstractManifold,
+        TO<:GeneralUnitaryMultiplicationGroup,
+        TAD<:ActionDirection,
+    } <: AbstractGroupAction{TAD}
+
+Action of the (special) unitary or orthogonal group [`GeneralUnitaryMultiplicationGroup`](@ref)
+of type `On` columns of points on a matrix manifold `M`.
+
+# Constructor
```

Comment:
A matrix is the standard representation for shape spaces. I think it's fine. Notice that it's not just a power manifold because of additional constraints.